fe/src/main/java/org/apache/doris/load/routineload/KafkaProgress.java

```diff
@@ -57,7 +58,7 @@ public void setPartitionIdToOffset(Map<Integer, Long> partitionIdToOffset) {
     public void update(RoutineLoadProgress progress) {
         KafkaProgress newProgress = (KafkaProgress) progress;
         newProgress.getPartitionIdToOffset().entrySet().parallelStream()
-                .forEach(entity -> partitionIdToOffset.put(entity.getKey(), entity.getValue()));
+                .forEach(entity -> partitionIdToOffset.put(entity.getKey(), entity.getValue() + 1));
```

Comment:
Why + 1？

Reply:
The offset of txn is the end offset while the kafka offset of progress is begin offset . The next task will read data from this begin offset.